### PR TITLE
Add Ruby 2.5 support in AWS Lambda

### LINKED ISSUE #5540: Ruby support
# This is a Feature Proposal

## Description

With the release of Ruby as a supported run-time for lambda it would be great to have support for it in serverless.

https://aws.amazon.com/blogs/compute/announcing-ruby-support-for-aws-lambda/

Similar or dependent issues:
* #5476 might be related, but seems to be asking about something different than the native Ruby support that AWS just announce.

### PR DESCRIPTION
**Pull request by BigChief45**
https://aws.amazon.com/about-aws/whats-new/2018/11/aws-lambda-supports-ruby/

Not sure if there are things that have to be added to docker-compose.yml.

## What did you implement:

Closes #5540

## How can we verify it:

```
serverless create --template aws-ruby
serverless deploy
serverless invoke -f hello
```

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

**Comment by dschep** [comment]
I already had a PR over at #5544 but this is more complete! Thanks 👍

**Comment by dschep** [comment]
Everything look good here @horike37 ?

File: lib/plugins/create/templates/aws-ruby/.gitignore [hunk 1]
```diff
@@ -0,0 +1,50 @@
+*.gem
```
Comment (dschep) [inline lib/plugins/create/templates/aws-ruby/.gitignore]: Actually.. counter to what @horike37 mentioned on my PR. I think this should be `gitignore` not `.gitignore`. Look at all the other runtime templets 😉

Comment (PedroSeda) [reply lib/plugins/create/templates/aws-ruby/.gitignore]: https://github.com/serverless/serverless/blob/b04473bac1410c01aa1a8c1bdf2b82040f91d6ca/lib/plugins/create/create.js#L220-L222
I agree with @dschep on this one.
Other then that, great work @BigChief45, looking forward to use it.

**Comment by rupakg** [comment]
Good work everyone!

**Pull request merged**